examples/oracle.yaml

```diff
@@ -0,0 +1,16 @@
+arch: "x86_64"
+
+images:
+- location: "https://yum.oracle.com/templates/OracleLinux/OL8/u5/x86_64/OL8U5_x86_64-olvm-b113.qcow2"
```

Comment:
Where to find the latest image?
Also, please link the license https://www.oracle.com/downloads/licenses/oracle-linux-license.html

Reply:
Googled "OL8U5_x86_64-olvm-b113.qcow2" but no result.

I guess this URL isn't expected to be directly linked?

Reply:
Will add license. I just used the links listed here: https://yum.oracle.com/oracle-linux-templates.html

Reply:
Yes, comment in the YAML is probably fine

Reply:
License: https://github.com/lima-vm/lima/pull/700/files#diff-ec074ef274eaabe9a71610791c8ed66c88b59ba7bb648390f2c34426d8130a07R1